tests/test_framework/phpbb_functional_test_case.php

```diff
@@ -911,10 +911,15 @@ static public function assert_response_xml($status_code = 200)
 	* status code. This assertion tries to catch that.
 	*
 	* @param int $status_code	Expected status code
-	* @return null
+	* @return void
 	*/
 	static public function assert_response_status_code($status_code = 200)
 	{
+		if ($status_code != self::$client->getResponse()->getStatus() &&
+			preg_match('/^5[0-9]{2}/', self::$client->getResponse()->getStatus()))
+		{
+			self::fail('Encountered unexpected server error:\n' . self::$client->getResponse()->getContent());
```

Comment:
actually we might want to keep that

Reply:
Tracker ticket is wrong, that's the second case I noticed, takes always to "12345".